### ISSUE TITLE
Notes RSS invalid

### ISSUE THREAD

**Issue by tomhughes** [tomhughes/trac-tickets#4862]
**[Submitted to the original trac issue database at 6.21am, Friday, 24th May 2013]**

Hi, even if Firefox displays the notes RSS feed, it seem to be bad formated.
I use TTRSS and get the message "notes: MagpieRSS: Failed to parse RSS file. (Not well-formed (invalid token) at line 1, column 1)"
I use this feed URL to monitor my state:
http://api.openstreetmap.org/api/0.6/notes/feed?bbox=10.79,53.15,14.39,54.61

Please fix it to allow more users to watch for notes bug reports :)

**Comment by tomhughes** [comment]
**Author: TomH**
**[Added to the original trac issue at 7.16am, Friday, 24th May 2013]**

My Firefox doesn't seem to have any problem with it, and line 1 column 1 is this:

<?xml version="1.0" encoding="UTF-8"?>

which seems to be a perfectly well formed XML declaration to me.

**Comment by tomhughes** [comment]
**Author: Andy Allan**
**[Added to the original trac issue at 7.46am, Friday, 24th May 2013]**

The original report is that it works with Firefox but not another program, so it's unsurprising that you found it works with Firefox!

The feed is invalid according to the w3c validator, and perhaps the other program (TTRSS) is just giving an unhelpful output from its XML parser.

http://validator.w3.org/feed/check.cgi?url=http%3A%2F%2Fapi.openstreetmap.org%2Fapi%2F0.6%2Fnotes%2Ffeed%3Fbbox%3D10.79%2C53.15%2C14.39%2C54.61

The two errors with the feed are:

```
line 56, column 21: Invalid email address: Noframe (8 occurrences) [help]

          <author>Noframe</author>

line 129, column 57: guid values must not be duplicated within a feed: http://api.openstreetmap.org/browse/note/3527 (4 occurrences) [help]

          <guid>http://api.openstreetmap.org/browse/note/3527</guid>
```

**Comment by tomhughes** [comment]
**Author: TomH**
**[Added to the original trac issue at 8.01am, Friday, 24th May 2013]**

It says "even if Firefox displays the notes RSS feed, it seem to be bad formated" which to me at least read as an assertion that it didn't appear correctly in Firefox.

**Comment by tomhughes** [comment]
**Author: TomH**
**[Added to the original trac issue at 8.24am, Friday, 24th May 2013]**

Worked around bogus RSS author element specification in [changeset:fa95768/rails] and fixed the duplicate IDs in [changeset:692d577/rails].
